cmd/daytona/config/const.go

```diff
@@ -90,6 +93,8 @@ func GetScopesFromGitProvider(providerId string) string {
 		fallthrough
 	case "gitea":
 		return "read:organization,write:repository,read:user"
+	case "gitness":
```

Comment:
Don't gitness tokens have scopes that should be applied?

Reply:
no gitness don't have token scopes.

Reply:
Done